src/libsyncengine/update_detection/file_system_observer/snapshot/snapshot.cpp

```diff
@@ -69,16 +69,14 @@ bool Snapshot::updateItem(const SnapshotItem &newItem) {
     if (newItem.parentId().empty()) {
         LOG_WARN(Log::instance()->getLogger(), "Parent ID is empty for item " << newItem.id().c_str());
         assert(false);
-        return false;
```

Comment:
For non-debug build types, the execution will continue without returning `false` immediately. Is it intended?

Reply:
Done